lib/metadataTypes/definitions/Interaction.definition.js

```diff
@@ -128,9 +128,9 @@ module.exports = {
         },
         modifiedDate: {
             isCreateable: false,
-            isUpdateable: false,
+            isUpdateable: true,
             retrieving: true,
-            template: false,
+            template: true,
```

Comment:
explain?

Reply:
if it's needed for update, we need to include it for templates